standalone/test/unit_tests/api/unittest_laserscan.cpp

```diff
@@ -157,24 +159,40 @@ TEST(LaserScanTest, testGetActiveZoneset)
   EXPECT_EQ(DEFAULT_ACTIVE_ZONESET, laser_scan->getActiveZoneset());
 }
 
+TEST(LaserScanTest, testSetAndGetIOStates)
+{
+  LaserScanBuilder laser_scan_builder;
+  std::unique_ptr<LaserScan> laser_scan;
+  ASSERT_NO_THROW(laser_scan.reset(new LaserScan(laser_scan_builder.build())););
+
+  laser_scan->setIOStates({ DEFAULT_IO_STATE });
+  EXPECT_EQ(laser_scan->getIOStates()[0].logicalInput(), DEFAULT_IO_STATE.logicalInput());
+  EXPECT_EQ(laser_scan->getIOStates()[0].output(), DEFAULT_IO_STATE.output());
+}
+
 TEST(LaserScanTest, testPrintMessageSuccess)
 {
   LaserScanBuilder laser_scan_builder;
   std::unique_ptr<LaserScan> laser_scan;
   ASSERT_NO_THROW(laser_scan.reset(new LaserScan(laser_scan_builder.build())););
 
   laser_scan->setMeasurements({ 45.0, 44.0, 43.0, 42.0 });
+  laser_scan->setIOStates({ IOState({ PinState(3, "io_pin_data", true) }, {}) });
```

Comment:
Looking at this calls for a API allowing

```suggestion
  laser_scan->setIOStates({ PinState(3, "io_pin_data", true) }, {});
```

Reply:
Don't you think a method `setIOStates` which gets only a single `IOState` is confusing? If you add one more pair of braces, it should already work.